binderhub-service/templates/builder-network-policy.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
let's put this behind a 'networkpolicy.enabled' flag or similar, and default it to true (same as whatever z2jh does)